graphics/BUILD.gn

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//flutter/common/config.gni")
+import("//flutter/testing/testing.gni")
+
+source_set("graphics") {
+  sources = [
+    "gl_context_switch.cc",
+    "gl_context_switch.h",
+    "persistent_cache.cc",
+    "persistent_cache.h",
+    "texture.cc",
+    "texture.h",
+  ]
+
+  deps = [
```

Comment:
Since we don't have any enforcement mechanism, we should at least have a big obvious comment here communicating that arbitrary dependencies should *not* be added here.

Reply:
Makes sense. Will do.